### ISSUE TITLE
Harman Air SDK website

### ISSUE THREAD

**Issue by rewb0rn** [Gamua/Adobe-Runtime-Support#91]
More and more people are asking how and where to get access to the Air SDK 33. Any news on the website from Harman?

**Comment by ajwfrost** [comment]
Little bits of progress ... we now have all the legal paperwork completed that will allow us to use the appropriate trademarks in ways that they're not usually permitted; we have had to update our plans a little to ensure compliance for data privacy and GDPR etc, and are going through another review for that. Marketing review was signed off now so that's something at least!

It's all taking longer than we had anticipated, and in the meantime there's a lot of manual effort going into the communications and sending out the SDK etc, so the sooner we can get this live, the better it will be for all of us!

thanks
